src/components/Dashboard/ThisWeek/WeeklyActivity/WeeklyReportModal/WeeklyReport.mock.ts

```diff
@@ -0,0 +1,71 @@
+import { MockedResponse } from '@apollo/client/testing';
+import {
+  CurrentCoachingAnswerSetDocument,
+  CurrentCoachingAnswerSetQuery,
+} from './WeeklyReportModal.generated';
+
+const populatedCoachingAnswerSet: CurrentCoachingAnswerSetQuery = {
+  currentCoachingAnswerSet: {
+    questions: [
+      {
+        id: 'question-1',
+        prompt: 'What is your goal for this week?',
+        required: true,
+        position: 1,
+      },
+      {
+        id: 'question-2',
+        prompt: 'Rate your progress on a scale of 1 to 5.',
+        required: true,
+        position: 2,
+      },
+      {
+        id: 'question-3',
+        prompt: 'What challenges did you face?',
+        required: true,
+        position: 3,
+      },
+    ],
+    answers: [],
+    id: 'answer-set-123',
+    completedAt: null,
+  },
+};
+
+const emptyCoachingAnswerSet: CurrentCoachingAnswerSetQuery = {
+  currentCoachingAnswerSet: {
+    questions: [],
+    answers: [],
+    id: 'answer-set-123',
+    completedAt: null,
+  },
+};
+
+export const CurrentCoachingAnswerSetDefaultMocks = (): MockedResponse[] => {
```

Comment:
Thanks for figuring out the weirdness with the mocks! Since a test will never want to use both of these mocks, can you split them into two separate mock exports and give them descriptive names like `populateCoachingAnswerSetMock` and `emptyCoachingAnswerSetMock`? You can make each of them a mock object - no need for them to be an array.